src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.IntegrationAccount.tests.ps1

```diff
@@ -406,5 +406,150 @@ InModuleScope Arcus.Scripting.IntegrationAccount {
                 }
             }
         }
+        Context "Handling Assemblies" {
```

Comment:
Maybe some extra description here? 😅 IDK, something like `Setting Azure Integration Account assemblies`.

Reply:
Changed it to `Uploading Assemblies into an Azure Integration Account`, also changed it for Schemas and Maps to keep it consistent.